### PR TITLE
Add afternoon feature

### PR DESCRIPTION
**Pull request by orkhan-huseyn**

### CODE REVIEW

**Comment by hazratahmdzd** [comment]
salam men commentem

**Review by hazratahmdzd — COMMENTED** [review]
men de commented

File: index.js [hunk 1]
```diff
@@ -9,3 +9,7 @@ function xumaritWorld() {
 function goodbyeWorld() {
    console.log('Goodbye, World!');
 }
+
+function goodAfternoonWorld() {
+  console.log('Good afternoon, World!');
+}
```
Comment (orkhan-huseyn) [inline index.js:15]:
```suggestion
function goodAfternoonWorld() {
  console.log('Good Afternoon, World!');
}
```

File: index.js [hunk 2]
```diff
@@ -9,3 +9,7 @@ function xumaritWorld() {
 function goodbyeWorld() {
    console.log('Goodbye, World!');
 }
```
Comment (eshginfarzali) [inline index.js:11]:
```suggestion
function goodbyeWorld() {
   console.log('Goodbye, World!');
} birinci herfi boyuk yaz
```

**Review by eshginfarzali — APPROVED** [review]

File: index.js [hunk 3]
```diff
@@ -9,3 +9,7 @@ function xumaritWorld() {
 function goodbyeWorld() {
    console.log('Goodbye, World!');
```
Comment (hazratahmdzd) [inline index.js:10]: what the hell?

**Review by hazratahmdzd — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (orkhan-huseyn) [reply index.js:15]: He yaxshi qalsin.

File: index.js (hunk 2 above)
Comment (orkhan-huseyn) [reply index.js:11]: Bunun neye xeyri var?

File: index.js (hunk 3 above)
Comment (orkhan-huseyn) [reply index.js:10]: 🤕

**Review by hazratahmdzd — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (orkhan-huseyn) [inline index.js:15]: asdasd

**Pull request merged**